assets/js/googlesitekit-i18n.js

```diff
@@ -0,0 +1,38 @@
+/**
+ * Site Kit i18n.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {
+	sprintf,
+	setLocaleData,
+	__,
+	_x,
+	_n,
+	_nx,
+	isRTL,
+} from '@wordpress/i18n__non-shim';
+
+global.wp = global.wp || {};
+global.wp.i18n = global.wp.i18n || {
```

Comment:
We shouldn't use the `wp.i18n` global as it might conflict with the code shipped and possibly loaded by WordPress, which is explicitly why this was mentioned to be solved differently in the IB. Let's follow the IB and define this at a separate `googlesitekit.i18n` which is fully controlled by us.